hl7/4.2/modules/ROOT/edits/hl7-connector-config-topics.adoc

```diff
@@ -0,0 +1,269 @@
+
+* If the MSG-01 Message Code value is `ACK`, always use the predefined ACK schema.
+* If the MSG-03 Message Structure value is present (a value such
+as `ADT_A01`), use the schema structure with that ID.
+* If neither of these, then use a configured Event and Message to Message Structure Map to determine
+the message structure from the specified Trigger Event (`MSG-02`) and Message Code (`MSG-01`) values.
+
+The Event and Message to Message Structure Map is an optional configuration parameter.
+It must be a YAML file consisting of a map from each Event Type to a map for each
+supported Message Code to the actual Message Structure.
+
+Here's an example:
+
+[source,yaml,linenums]
+----
+A01: { XYZ: ADT_A01, ACK: ACK }
+A02: { XYZ: ADT_A02, ACK: ACK }
+A03: { XYZ: ADT_A03, ACK: ACK }
+A04: { XYZ: ADT_A01, ACK: ACK }
+A05: { XYZ: ADT_A05, ACK: ACK }
+A06: { XYZ: ADT_A06, ACK: ACK }
+A07: { XYZ: ADT_A06, ACK: ACK }
+A08: { XYZ: ADT_A01, ACK: ACK }
+----
+
+Each version of HL7 defines a different set of mappings from the event type and
+message code to the message structure. The default mappings are provided in the
+same JAR as the standard HL7 schema definitions, in files named `event-message.yaml`.
+You use the same type of paths for these mapping definitions as for the actual message structure schemas.
+
+=== General Tab
+
+Use the *General* tab to configure settings for reading and writing HL7 messages:
+
+* HL7 character encoding, always used for writing messages and used
+when reading messages unless a different encoding is specified by MSH-18 (Character Set).
+* Disable numeric prefixes for data keys - this option is true by
+default, which turns off numeric prefixes for segment data. The
+only reason to turn this option off is for compatibility with mappings
+defined for the HL7 EDI Connector 3.0.0.
+* Manually create or edit the list of schemas.
+
+=== Set Your HL7 Identification in the Visual Editor
+
+You can configure the Message Header (MSH) application and facility
+identification for you and your trading partner on the HL7 EDI connector configuration.
+
+The values you set are used when writing HL7 messages to supply the
+namespace ID, universal ID, and universal ID type, and are verified
+in receive messages. If you don't want to restrict incoming messages
+then leave these blank, and set the values for outgoing messages
+on the write operation or the actual outgoing message. Values set on
+the write operation override the connector configuration, and values
+set directly on the message override both the connector configuration
+and any values set on the write operation.
+
+In Studio, these values are set in these Global Element Properties.
+
+* Self identification parameters identify your side of the trading partner relationship.
++
+Self identification settings:
```

Comment:
Do not use fragments like this one. Either make this a complete sentence or incorporate it into the sentence above.
Per style ref.